src/ert/enkf_main.py

```diff
@@ -215,7 +216,7 @@ def loadFromForwardModel(
         t = time.perf_counter()
         run_context = self.ensemble_context(fs, realization, iteration)
         nr_loaded = fs.load_from_run_path(
-            self.getEnsembleSize(),
```

Comment:
Are we guaranteed that ensemble size is always set in storage here?

Reply:
Yes, but will split this into another commit, or perhaps PR, dont have to do that here.